airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -163,6 +164,8 @@ def __init__(self, context: ConnectorContext) -> None:
             context=context,
             paths_to_mount=[
                 MountPath(code_directory),
+                # Support airbyte-enterprise docs.
+                MountPath(SUBMODULE_ENTERPRISE_DOCS_PATH),
```

Comment:
I'm not sure how/if this mount path is symlinked to the main doc path.
The way [we define the `connector.documentation_file_path` attribute is by:](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/connector_ops/connector_ops/utils.py#L306)
* Keeping the path portion after `https//docs.airbyte.com/<the path relative to .docs>` 
* Appending it to `./docs`

If the symlink works I think we're good.

Reply:
the symlink works in dagger if both the symlink and its target are mounted